inputfiles/idlSources.json

```diff
@@ -269,5 +269,9 @@
     {
         "url": "https://html.spec.whatwg.org/multipage/web-messaging.html",
         "title": "Web Messaging"
+    },
+    {
+        "url": "https://streams.spec.whatwg.org/",
+        "title": "Streams"
```

Comment:
Can you move this above Storage? Also, `npm run fetch` will fail because the spec page does not have any IDL. We may add `"local": true` and fix fetcher.ts so that it won't fetch IDL when marked as `local`.

Reply:
I believe you mean *below* storage, since `"Storage"` < `"Streams"`? Sure, I can move "Streams" into alphabetical order.